pkg/sidecar/sidecar_test.go

```diff
@@ -0,0 +1,139 @@
+package sidecar
+
+import (
+	"context"
+	"github.com/stretchr/testify/assert"
+	"github.com/testground/sdk-go/runtime"
+	"github.com/testground/sdk-go/sync"
+	"math/rand"
+	"strconv"
+	"testing"
+	"time"
+)
+
+func init() {
+	rand.Seed(time.Now().UnixNano())
+}
+
+const (
+	CONFIGUREDNETWORK   = "testtesttest_net"
+	CONFIGUREDBANDWIDTH = uint64(12345)
+	NETWORKDONE         = "configureddone"
+)
+
+func unique_runenv() (hostname string, runenv *runtime.RunEnv) {
```

Comment:
Can't we just use https://github.com/testground/sdk-go/blob/207954a995a3933d3c55a0466e483491ffa78e1d/runtime/test_utils.go#L14?